src/styles/dist/landing.css

```diff
@@ -612,30 +612,12 @@ footer {
   color: #003b25;
 }
 
-
-/* About Us */
-.about-us{
-  display: flex;
-  justify-content: center;
-  align-items: center;
-}
-
-.about-us-body{
-  text-align: center;
-  padding: 5%;
-}
-
-.connect-with-us{
-  width: 25px;
-  margin: 2%;
-}
-
-
 /* About Us */
 .about-us{
   display: flex;
   justify-content: center;
   align-items: center;
+  background-color: #003b25;
```

Comment:
Can I confirm if in the about page the background really should be green? Just asking upon referring to figma

![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/92857477/65dbf7af-9a39-4c1e-9480-fc98b28bc88c)